include/CSVManager.h

```diff
@@ -0,0 +1,12 @@
+#include<vector>
+#include<string>
+
+class CSVManager {
```

Comment:
casbin doesn't read csv file directly, it uses an adapter to read policy, csv file is a kind of adapter.